npmRunSpuild failed for all builds

All builds are failing in both Travis and AppVeyor.

```
Starting a Gradle Daemon (subsequent builds will be faster)
> Task :downloadNode
Trying to download http://mirrors.lax.liferay.com/nodejs.org/dist/v10.5.0/node-v10.5.0-linux-x64.tar.gz to /home/travis/.liferay/mirrors/nodejs.org/dist/v10.5.0/node-v10.5.0-linux-x64.tar.gz.tmp
Trying to download http://nodejs.org/dist/v10.5.0/node-v10.5.0-linux-x64.tar.gz to /home/travis/.liferay/mirrors/nodejs.org/dist/v10.5.0/node-v10.5.0-linux-x64.tar.gz.tmp
> Task :npmInstall
npm WARN deprecated flatten@1.0.2: I wrote this module a very long time ago; you should use something else.
npm WARN deprecated browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
npm WARN deprecated circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
> core-js@2.6.9 postinstall /home/travis/build/reposense/RepoSense/node_modules/core-js
> node scripts/postinstall || echo "ignore"
npm WARN acorn-dynamic-import@4.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.0.7 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.0.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
added 958 packages from 590 contributors and audited 3522 packages in 17.564s
found 2 vulnerabilities (1 moderate, 1 high)
  run `npm audit fix` to fix them, or `npm audit` for details
> Task :npmRunSpuild FAILED
> reposense@1.0.0 spuild /home/travis/build/reposense/RepoSense
> spuild frontend
frontend/src/favicon.ico
frontend/src/index.pug
frontend/src/ramp.pug
frontend/src/static/css/_colors.scss
undefined:3
throw error;
^
Error: NoSuchMethodError: method not found: 'pathToFileURL' (url.pathToFileURL is not a function)
    at Object._newRenderError (/home/travis/build/reposense/RepoSense/node_modules/sass/sass.dart.js:10677:19)
    at StaticClosure._renderSync (/home/travis/build/reposense/RepoSense/node_modules/sass/sass.dart.js:10506:18)
    at Object.Primitives_applyFunction (/home/travis/build/reposense/RepoSense/node_modules/sass/sass.dart.js:1045:30)
    at Object.Function_apply (/home/travis/build/reposense/RepoSense/node_modules/sass/sass.dart.js:4872:16)
    at _callDartFunctionFast (/home/travis/build/reposense/RepoSense/node_modules/sass/sass.dart.js:6477:16)
    at Object.renderSync (/home/travis/build/reposense/RepoSense/node_modules/sass/sass.dart.js:6455:18)
    at scss (/home/travis/build/reposense/RepoSense/node_modules/spuild/index.js:23:15)
    at files.forEach (/home/travis/build/reposense/RepoSense/node_modules/spuild/index.js:83:32)
    at Array.forEach (<anonymous>)
    at glob (/home/travis/build/reposense/RepoSense/node_modules/spuild/index.js:69:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! reposense@1.0.0 spuild: `spuild frontend`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the reposense@1.0.0 spuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-07-25T03_44_46_421Z-debug.log
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':npmRunSpuild'.
> Process 'command '/home/travis/build/reposense/RepoSense/build/node/bin/node'' finished with non-zero exit value 1
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 35s
3 actionable tasks: 3 executed
```

--- a/build.gradle
+++ b/build.gradle
@@ -12,7 +12,7 @@ plugins {
 
 mainClassName = 'reposense.RepoSense'
 
-node.nodeVersion = '10.5.0'
+node.nodeVersion = '10.16.0'
 
 sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8

